lesson4/calendar.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html>
+  <head lang="en">
+    <title>Schedule</title>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <link rel="stylesheet" href="styles_calendar.css" />
+  </head>
+  <body>
+    <div class="calendar">
+      <div class="calendar_header">
+        <div id="month_prev" class="calendar_month"></div>
+        <div id="month_curr" class="calendar_month"></div>
+        <div id="month_next" class="calendar_month"></div>
+      </div>
+      <div class="week_days">
```

Comment:
Let's moved it in JS, because, in real project are a lot of languages then filling of all visible for user strings in the JS file is the best practice.